src/olympia/browse/urls.py

```diff
@@ -52,8 +52,10 @@
         ThemeCategoriesRss(), name='browse.themes.rss'),
 
     # This won't let you browse any themes but detail page needs the url.
-    url('^static-themes/(?P<category>[^/]+)?$', views.extensions,
-        name='browse.static-themes'),
+    url('^static-themes/(?P<category>[^/]+)?$',
+        lambda r, category: redirect(
+            reverse('browse.personas', kwargs={
+                'category': category or ''})), name='browse.static-themes'),
```

Comment:
That's quite a lambda!  There's an argument for making it a full view method and doing the redirection in there but it works - I did a quick test locally.
I think
```
    url('^static-themes/(?P<category>[^/]+)?$',
        lambda r, category: redirect(
            reverse('browse.personas', kwargs={'category': category or ''})),
        name='browse.static-themes'),
```
reads a little clearer though.